src/Distributions.cpp

```diff
@@ -46,7 +46,7 @@ DISTRIBUTION(mean_norm,
 
 DISTRIBUTION(var_norm,
 
-    inline static std::string description = "Normal Distribution with change in Variance and constant Mean";
+    static std::string description;
```

Comment:
@diego-urgell can this be moved to the class definition so that it does not need to be repeated for each distribution?

Reply:
This was later moved to the RcppInterface file, because of the C++17 problems.